.github/workflows/ci.yaml

```diff
@@ -25,23 +34,26 @@ jobs:
           echo "PLATFORM_OS=$(echo ${{ matrix.platform }} |  cut -d/ -f1)" >> $GITHUB_OUTPUT
           echo "PLATFORM_ARCH=$(echo ${{ matrix.platform }} |  cut -d/ -f2)" >> $GITHUB_OUTPUT
 
-      - name: checkout repo
-        uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b # pin@v3
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
 
       - name: Set up QEMU
-        uses: docker/setup-qemu-action@e81a89b1732b9c48d79cd809d8d81d79c4647a18 #pin@v2.1.0
+        uses: docker/setup-qemu-action@v2
```

Comment:
Please don't undo work of others #9 

GH action version pinning to hash is here for a reason. I'm not sure how unpinning them is making the build any faster (as per the PR title). It definitely makes it less secure.

https://michaelheap.com/ensure-github-actions-pinned-sha/
https://exercism.org/docs/building/github/gha-best-practices#h-pin-actions-to-shas